css/about.css

```diff
@@ -0,0 +1,155 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
If you want you can also take the normalize/reset CSS code and put it in its own file as well.  That way you don't have to copy and paste it from the index.css.